native_words.asm

```diff
@@ -3175,6 +3144,12 @@ do_runtime:
         ; However, we can do it faster if we just copy the bytes
         ; of this routine with a simple loop in DO.
         ; """
+                ; stash return address so we can manipulate return stack
```

Comment:
slightly more expensive to call this as a subroutine, but it only happens on loop startup and saves a bunch of space on every use of DO

Reply:
As you already mentioned, the DO/LOOP system is already heavy in both bytes and cycles.  I think this is a good tradeoff.  It might be worth adding this to the debugger dispatch table - alternatively, it could be added to the dictionary as something like `(DO)`.

Reply:
i'm not sure what you mean when you say debugger dispatch table.  is that so it can appear labeled in disasm or something else?

Reply:
Yes, so it will be labeled in the disassembler (which is what I meant when I wrote debugger - my bad) - otherwise it will be an unlabeled JSR and might not be clear that it's the beginning of a loop.